### PR TITLE
Add scripts to create the humble image

### PR DESCRIPTION
**Pull request by carlossvg**
**Note: congi.ini has to be updated:**

```diff
-image_size = 4G
+image_size = 8G
```

### CODE REVIEW

**Comment by razr** [comment]
@carlossvg I have a feeling this repo shall be merged into https://github.com/ros-realtime/ros-realtime-rpi4-image

how about creating a `configs` directory and putting this configuration inside? Then we can use already existing GitHub actions to build benchmarks image

File: ros2-rt-benchmark-tools/scripts/phase1-target [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+# init rosdep
+rosdep init
+rosdep update
+
+# install dependencies for reference-system
+python3 -m pip install psrecord bokeh
+
+mkdir -p ~/ros2_rt_benchmarks_ws/src
+cd ~/ros2_rt_benchmarks_ws/src
```
Comment (LanderU) [inline ros2-rt-benchmark-tools/scripts/phase1-target]:
```suggestion
cd ~/ros2_rt_benchmarks_ws/src || exit 1
```

File: ros2-rt-benchmark-tools/scripts/phase1-target [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+git clone https://gitlab.com/ApexAI/performance_test.git
```
Comment (LanderU) [inline ros2-rt-benchmark-tools/scripts/phase1-target:41]: Branch?

File: ros2-rt-benchmark-tools/scripts/phase1-target [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+git clone https://github.com/ros-realtime/reference-system
```
Comment (LanderU) [inline ros2-rt-benchmark-tools/scripts/phase1-target:42]: Branch?

File: ros2-rt-benchmark-tools/scripts/phase1-target [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+cd ..
```
Comment (LanderU) [inline ros2-rt-benchmark-tools/scripts/phase1-target]:
```suggestion
cd .. || exit 1
```

File: scripts/phase1-target [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+# init rosdep
+rosdep init
+rosdep update
+
+# install dependencies for reference-system
+python3 -m pip install psrecord bokeh
+
+mkdir -p ~/ros2_rt_benchmarks_ws/src
+cd ~/ros2_rt_benchmarks_ws/src
```
Comment (LanderU) [inline scripts/phase1-target:40]:
```suggestion
cd ~/ros2_rt_benchmarks_ws/src || exit 1
```

File: scripts/phase1-target [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+git clone https://gitlab.com/ApexAI/performance_test.git
```
Comment (LanderU) [inline scripts/phase1-target:41]: Branch?

File: scripts/phase1-target [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+git clone https://github.com/ros-realtime/reference-system
```
Comment (LanderU) [inline scripts/phase1-target:42]: Branch?

File: scripts/phase1-target [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+cd ..
```
Comment (LanderU) [inline scripts/phase1-target:43]:
```suggestion
cd .. || exit 1
```

File: ros2-rt-benchmark-tools/scripts/phase1-target [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+rosdep install --from-paths src --ignore-src -y --skip-keys "fastcdr rti-connext-dds-6.0.1 urdfdom_headers"
+
+# build tools
+source /opt/ros/$ROS_DISTRO/setup.bash
+colcon build --cmake-force-configure --cmake-args -DRUN_BENCHMARK=ON
```
Comment (LanderU) [inline ros2-rt-benchmark-tools/scripts/phase1-target]: Do we want to configure the install path as well?

**Comment by carlossvg** [comment]
> @carlossvg I have a feeling this repo shall be merged into https://github.com/ros-realtime/ros-realtime-rpi4-image

The reason I created this in a separate repo is because I though this might not be relevant for other use cases. I don't have a strong opinion about this.

I would like to know @shuhaowu 's input on this.

**Review by LanderU — APPROVED** [review]
LGTM!

**Comment by shuhaowu** [comment]
> > @carlossvg I have a feeling this repo shall be merged into https://github.com/ros-realtime/ros-realtime-rpi4-image
> 
> The reason I created this in a separate repo is because I though this might not be relevant for other use cases. I don't have a strong opinion about this.
> 
> I would like to know @shuhaowu 's input on this.

I don't really have a strong opinion about this either. Either way is fine by me. 

Also, where's the 8G config.ini change? I don't see it?

**Comment by carlossvg** [comment]
>Also, where's the 8G config.ini change? I don't see it?

Because it's this one and it's part of the other repo: https://github.com/ros-realtime/ros-realtime-rpi4-image/blob/master/image_builder/data/jammy-rt/config.ini#L12

**Comment by carlossvg** [comment]
@shuhaowu @razr One of the reasons I created a separate repo was to upload the images in a different place. I'm afraid that if we start adding many different images it could be confusing for users just looking for a ROS 2 image.

**Comment by shuhaowu** [comment]
> > Also, where's the 8G config.ini change? I don't see it?
> 
> Because it's this one and it's part of the other repo: https://github.com/ros-realtime/ros-realtime-rpi4-image/blob/master/image_builder/data/jammy-rt/config.ini#L12

You should be able to override it from the `config.ini` file here, unless that is broken for some reason?

**Comment by carlossvg** [comment]
>You should be able to override it from the config.ini file here, unless that is broken for some reason?

Ah, that's true. I added it.

**Pull request merged**